front/src/modules/trainschedule/components/ManageTrainSchedule/Map.tsx

```diff
@@ -65,6 +71,20 @@ const Map = () => {
 
   const mapRef = useRef<MapRef | null>(null);
 
+  const getMapImageData = useCallback(() => {
```

Comment:
Do you really need a useCallback here ? Why not put everything in the useEffect ?

Reply:
done